src/descriptors/power_system_structs.json

```diff
@@ -1972,7 +1976,8 @@
             "max": null
           },
           "validation_action": "error",
-          "needs_conversion": true
+          "needs_conversion": true,
```

Comment:
Same here, these fields should have reasonable values

Reply:
Removed